vector-stores/spring-ai-qdrant-store/src/test/java/org/springframework/ai/vectorstore/qdrant/QdrantVectorStoreBuilderTests.java

```diff
@@ -70,14 +70,14 @@ void customConfiguration() {
 	@Test
 	void nullQdrantClientInConstructorShouldThrowException() {
 		assertThatThrownBy(() -> QdrantVectorStore.builder(null, null)).isInstanceOf(IllegalArgumentException.class)
-			.hasMessage("QdrantClient must not be null");
+			.hasMessage("EmbeddingModel must be configured");
 	}
 
 	@Test
 	void nullEmbeddingModelShouldThrowException() {
 		assertThatThrownBy(() -> QdrantVectorStore.builder(qdrantClient, null).build())
 			.isInstanceOf(IllegalArgumentException.class)
-			.hasMessage("EmbeddingModel must not be null");
+			.hasMessage("EmbeddingModel must be configured");
```

Comment:
This message now is defined in the abstract parent builder, so it differs from the one that used to be defined in QdrantVectorStore itself.